src/Workspaces/Core/Portable/Rename/HeavyweightRenameLocations.cs

```diff
@@ -48,74 +48,22 @@ internal sealed partial class RenameLocations
             Symbol = symbol;
             Options = options;
             FallbackOptions = fallbackOptions;
+            Contract.ThrowIfNull(locations);
             Locations = locations;
             ReferencedSymbols = referencedSymbols;
             ImplicitLocations = implicitLocations;
         }
 
-        internal static RenameLocations Create(
-            ImmutableHashSet<RenameLocation> locations,
-            ISymbol symbol,
-            Solution solution,
-            ImmutableArray<ISymbol> referencedSymbols,
-            ImmutableArray<ReferenceLocation> implicitLocations,
-            SymbolRenameOptions options,
-            CodeCleanupOptionsProvider fallbackOptions)
-        {
-            return new RenameLocations(
-                symbol, solution, options, fallbackOptions, locations, implicitLocations, referencedSymbols);
-        }
```

Comment:
no point in a factory method that just forwarded thnigs along.  NOte: the order of parameters was different too.  So you'll see several reference to this become a call to the construct, with some reordering.